templates/sc-swiper-card-product.php

```diff
@@ -29,25 +34,30 @@
 function bootscore_product_slider($atts) {
 
   ob_start();
-  extract(shortcode_atts(array(
-    'type' => 'product',
-    'order' => 'date',
-    'orderby' => 'date',
-    'posts' => -1,
-    'category' => '',
-    'featured' => '',
+  $atts = shortcode_atts(array(
+    'type'       => 'product',
+    'order'      => 'DESC',
+    'orderby'    => 'date',
+    'limit'      => 12,
```

Comment:
Why should we limit the amount to 12 by default instead show them all and let the user choose if he would to limit or not?

Reply:
It doesn't have to be 12, it can be more, but if the user doesn't properly configure the shortcode and they end up displaying all products, it's a massive query that can overload servers if they have hundreds of products. Plus after 80 products, the bullet navigation stacks on top of other elements. We should probably add an attribute to disable bullet navigation if the user does want to display more than 80 products and/or fix the bullet navigation style so it doesn't overlay other elements, though I doubt someone would want a massive bullet navigation spanning more than one line.

WooCommerce does set a -1 limit with their products shortcode, so we can leave it that way, but documentation should be added to make sure it's properly configured to narrow down the products to a category that has less than 80 products.